cli/command/container/hijack.go

```diff
@@ -71,45 +68,44 @@ func (h *hijackedIOStreamer) stream(ctx context.Context) error {
 	}
 }
 
-func (h *hijackedIOStreamer) setupInput() (restore func(), err error) {
-	if h.inputStream == nil || !h.tty {
+func setupInput(streams command.Streams, opts *HijackOptions) (restore func(), reader io.Reader, err error) {
```

Comment:
I think this should receive a non-pointer type for `opts`. It shouldn't be updating `opts`